app/src/main/res/layout/view_authorization_content.xml

```diff
@@ -0,0 +1,117 @@
+
+    For the additional permissions granted for Salt Edge Authenticator
+    under Section 7 of the GNU General Public License see THIRD_PARTY_NOTICES.md
+-->
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/shape_bg_rect_white_ripple"
+    android:clipToPadding="false"
+    android:transitionGroup="true"
+    android:padding="@dimen/dp_16">
+
+    <LinearLayout
+        android:id="@+id/contentLayout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginStart="@dimen/dp_16"
+        android:layout_marginEnd="@dimen/dp_16"
+        android:layout_marginTop="@dimen/dp_8"
+        android:layout_marginBottom="@dimen/dp_16"
+        android:layout_above="@+id/actionsLayout"
+        android:gravity="center_horizontal"
+        android:orientation="vertical"
+        tools:background="@android:color/holo_green_dark">
+
+        <TextView
+            android:id="@+id/titleTextView"
+            style="@style/Text16GrayDarkRegular"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:maxLines="2"
+            tools:background="@color/red"
+            tools:text="Create payment" />
+
+        <TextView
+            android:id="@+id/descriptionTextView"
+            style="@style/Text14GrayRegular"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_16"
+            android:gravity="start"
+            android:maxLines="6"
+            tools:background="@color/black" />
+
+        <WebView
+            android:id="@+id/descriptionWebView"
+            android:layout_marginTop="@dimen/dp_16"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:visibility="gone" />
+    </LinearLayout>
+
+    <ProgressBar
+        android:id="@+id/processingView"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_marginBottom="@dimen/dp_16"
+        android:indeterminate="true"
+        android:indeterminateTint="@color/blue"
+        android:indeterminateTintMode="src_in"
+        android:visibility="gone"/>
+
+    <LinearLayout
+        android:id="@+id/actionsLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_marginBottom="@dimen/dp_8"
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/negativeActionView"
```

Comment:
Please compare with `authorization_info`.
There is something to think about ;)

Reply:
please explain more clear

Reply:
i will remove authorization_info